crypto/Makefile

```
@@ -149,7 +150,19 @@ all: tools tests
 %.o: %.c %.h options.h
 	$(CC) $(CFLAGS) -o $@ -c $<
 
-tests: tests/test_check tests/test_openssl tests/test_speed tests/libtrezor-crypto.so tests/aestst
+# These Pallas units share pallas.h (no same-named header), so give them an
+# explicit rule that the pattern rule above would otherwise miss.
+pallas_poseidon.o: pallas_poseidon.c pallas.h pallas_poseidon_constants.h options.h
+	$(CC) $(CFLAGS) -o $@ -c $<
+pallas_drk.o: pallas_drk.c pallas.h options.h
+	$(CC) $(CFLAGS) -o $@ -c $<
+
+tests: tests/test_check tests/test_openssl tests/test_speed tests/test_pallas tests/libtrezor-crypto.so tests/aestst
```

checkmake (0.3.2)

[warning] Target "tests" should be declared PHONY.

(phonydeclared)